lib/Model/SelectHelper.php

```diff
@@ -50,6 +50,46 @@ public function selectRoomsTable(IQueryBuilder $query, string $alias = 'r'): voi
 		])->selectAlias($alias . 'id', 'r_id');
 	}
 
+	public function selectCommentsTable(IQueryBuilder $query, string $alias = 'c'): void {
+		if ($alias !== '') {
+			$alias .= '.';
+		}
+
+		$query->addSelect([
+			$alias . 'parent_id AS c_parent_id',
```

Comment:
using ` AS ` directly means the quoting for column names may not work properly?
Please use the selectAlias method?